tests/reactive_fluid_transport.prm

```diff
@@ -0,0 +1,155 @@
+# This is a 1D test case for an insoluble fluid moving upwards in a
+# narrow channel through porous coupled two-phase flow.
```

Comment:
The tester says: "test reactive_fluid_transport.prm is missing a folder with reference data"